src/config_builder.py

```diff
@@ -16,9 +16,12 @@
 
 LOKI_DIR = "/loki"
 CHUNKS_DIR = os.path.join(LOKI_DIR, "chunks")
+LOKI_CONFIG_PERSISTED = os.path.join(CHUNKS_DIR, "loki-local-config.yaml")
```

Comment:
Should we make it more obvious that this is a backup, for reference only?

```python
# Path to a persisted config backup, for reference purposes.
# This is needed to e.g. determine the storage backend used before an upgrade too place.
LOKI_CONFIG_BACKUP = os.path.join(CHUNKS_DIR, "loki-local-config.yaml.bak")
```

---

Off-topic:
This variable isn't used by the config builder but relies on some dir being persisted in metadata.yaml.
We should probably refactor these "consts" in a future PR.